### DISCUSSION TITLE
Line/Bar Charts & Pie Charts

### DISCUSSION THREAD

**Discussion by ovidiaconescu** [fbaligand/kibana-enhanced-table#232] [Q&A]
Hey, is there a way to take the data from a table that includes computed columns and use it in a different viz, like a Pie Chart or Line/Bar/Area chart?

**Comment by fbaligand** [comment]
Hi @ovidiaconescu,

Sorry, but no, it is not possible.

That said, a quite similar feature is Kibana Lens (where you can define Line/Bar/Area/Pie chart.
Actually, since Kibana 7.14, you can define "Formulas" that are quite equivalent to enhanced table "computed columns":
https://www.elastic.co/fr/blog/whats-new-kibana-7-14-0
